src/flamenco/types/fd_types.json

```diff
@@ -561,6 +561,15 @@
       ],
       "comment": "https://github.com/anza-xyz/agave/blob/7117ed9653ce19e8b2dea108eff1f3eb6a3378a7/sdk/src/reward_info.rs#L5"
     },
+    {
+      "name": "slot_lthash",
+      "type": "struct",
+      "archival": "true",
+      "alignment": "128",
+      "fields": [
+        { "name": "lthash",                      "type": "uchar[2048]" }
```

Comment:
Could we add a comment here indicating the type? So that people know to cast it to `fd_lthash_value_t`

Reply:
Better yet, we could actually make this a `fd_lthash_value_t`?

Reply:
This would require changes to the stub generator.  I will add a comment